appengine/pond_storage/common.py

```diff
@@ -183,7 +204,7 @@ def delete_duck(duck):
   duck.key.delete()
   return True
 
-def create_duck(name, code):
+def create_duck(name, opt_code):
```

Comment:
I think this needs a default value opt_code=??

Reply:
Good point, I've added a default value and removed the "opt_" because in looking at other examples it didn't seem like a common convention.

Reply:
I think you can still have the opt_ because technically someone doesn't need to pass it in. But up to you.